Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "scraper"
-version = "0.14.0"
+version = "0.14.1"
```

Comment:
We never edit version number in PRs. Version numbers are bumped when we cut and publish a new release.